target/linux/generic/hack-6.6/765-mxl-gpy-control-LED-reg-from-DT.patch

```diff
@@ -63,11 +63,11 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
 +	if (ret < 0)
 +		return ret;
 +
-+	eturn gpy_led_write(phydev);
++	return gpy_led_write(phydev);
```

Comment:
Following the error message, this line should be deleted.
66 - +  return gpy_led_write(phydev);

Reply:
OK ... should the new line inserted under ` 	/* Nothing to configure. Configuration Requirement Placeholder */` remain?

Reply:
> OK ... should the new line inserted under ` /* Nothing to configure. Configuration Requirement Placeholder */` remain?

Latest patch this line deleted:
https://github.com/openwrt/openwrt/pull/15818/files/19031bd08571aa8e7868d71d5e8d4ce45371d579..64ef2eb5080f580a048163942addcd8bd376c601#diff-8c7f9f802cfa866c19fb895d0477a1f743db82a320c06fa6c68bea69d1e78080R66